src/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Windows.cs

```diff
@@ -1094,29 +1084,22 @@ public byte[] PathTypes
                 return types;
             }
         }
-        public PointF[] PathPoints
+
+        public unsafe PointF[] PathPoints
         {
             get
             {
-                int count = PointCount;
-                int size = Marshal.SizeOf(typeof(GPPOINTF));
-                IntPtr buf = Marshal.AllocHGlobal(checked(count * size));
-                try
+                PointF[] points = new PointF[PointCount];
+                fixed(PointF* p = points)
```

Comment:
Nit: space before (

Reply:
D'oh, I searched and didn't click on this result. I'll get it in the next change.